pkg/lite-apiserver/proxy/proxy.go

```diff
@@ -173,9 +173,7 @@ func (p *EdgeReverseProxy) modifyResponse(res *http.Response) error {
 func (p *EdgeReverseProxy) handlerError(rw http.ResponseWriter, req *http.Request, err error) {
 	klog.Warningf("Request url %s, %s error %v", req.URL.Host, req.URL, err)
 	defer func() {
-		if p.watch {
```

Comment:
“DeleteRequest both when watch resource request fails and when list resource request fails.”
it is not safe to removing this.
it maybe put/post the same resource error.

Reply:
> e put/post the same resource error.

What do you mean by `it maybe put/post the same resource error.`?

Reply:
e.g. updating one node, the url is/api/v1/nodes/node***, verb is patch. it will delete the node list-watch if this patch request error.

Reply:
> e.g. updating one node, the url is/api/v1/nodes/node***, verb is patch. it will delete the node list-watch if this patch request error.

I have changed it back, and add update filterErrorToIgnore judge statement to filter those requests that do not need cache. PTAL